server/server.py

```diff
@@ -632,9 +637,18 @@ def handle_client_message(self, addr, message, room=None):
                         remaining_cooldown = 0
                         
                         if room.game.trains[nickname].boost_cooldown_active:
-                            current_time = time.time()
-                            elapsed_time = current_time - room.game.trains[nickname].start_cooldown_time
-                            remaining_cooldown = max(0, BOOST_COOLDOWN_DURATION - elapsed_time)
+                            # Use tick-based cooldown calculation
+                            current_tick = room.game.trains[nickname].move_timer
+                            ticks_elapsed = current_tick - room.game.trains[nickname].start_cooldown_tick
+                            
+                            # Convert duration to ticks using the same approach as in train.py
+                            standard_tickrate = 60.0  # Reference tickrate
```

Comment:
again, we can't hardcode 60.0 here and allow configuring the tick_rate in the client and server configs. We should honor the config options.